### PR TITLE
Issue #9400: update example of AST for TokenTypes.BLOCK_COMMENT_END

### LINKED ISSUE #9400: update example of AST for TokenTypes.BLOCK_COMMENT_END
https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#BLOCK_COMMENT_END

needs to be extended to show example of AST in format that printed by our CLI (change "+-" to "`-").
Source to be updated: https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

PLEASE READ for more details: https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-AST-in-TokenTypes

### PR DESCRIPTION
**Pull request by vivek286**
Closes #9400 

WhatsApp Image 2021-03-16 at 7 39 51 PM

Expected update example of AST for TokenTypes.BLOCK_COMMENT_END

WhatsApp Image 2021-03-16 at 7 40 25 PM

### CODE REVIEW

**Comment by vivek286** [comment]
@romani please review if you have time

**Comment by Abhishek-kumar09** [comment]
java Test.java
```java
class A {
  // hi
}
```
$ java -jar checkstyle-8.41-all.jar -t Test.java 
```
CLASS_DEF -> CLASS_DEF [1:0]
|--MODIFIERS -> MODIFIERS [1:0]
|--LITERAL_CLASS -> class [1:0]
|--IDENT -> A [1:6]
`--OBJBLOCK -> OBJBLOCK [1:8]
    |--LCURLY -> { [1:8]
    `--RCURLY -> } [3:0]
```
Comments are not getting any AST print, is it normal?

**Comment by vivek286** [comment]
Please guide me through the development IDE setup, the process given in documentation is a bit confusing.

**Comment by romani** [comment]
@Abhishek-kumar09 

> Comments are not getting any AST print, is it normal?

use `-T` CLI argument to print comment.

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -3544,16 +3544,14 @@
...
      * </pre>
      *
      * <pre>
-     * +--BLOCK_COMMENT_BEGIN
-     *         |
-     *         +--COMMENT_CONTENT
-     *         +--BLOCK_COMMENT_END
+     * |--BLOCK_COMMENT_BEGIN -&gt; BLOCK_COMMENT_BEGIN
+     *      |--COMMENT_CONTENT -&gt; COMMENT_CONTENT
+     *      |--BLOCK_COMMENT_END -&gt; BLOCK_COMMENT_END
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:3554]: please make java code examples to make it visual for user like `// comment` and `/* comment */`

**Review by romani — CHANGES_REQUESTED** [review]
item

**Comment by romani** [comment]
@vivek286 , please reply in next 7 days, if no reply, I will close PR

**Pull request closed without merging**